src/main/java/com/thejavatar/pokemongogo/PokemonDecorator.java

```diff
@@ -92,14 +95,72 @@ public int getEvolutionsPossible() {
         	return 0;
     }
     
+    public String getPokeball() {
+        if (ItemId.ITEM_MASTER_BALL == pokemon.getPokeball())
+        	return "Master Ball";
+        else if (ItemId.ITEM_ULTRA_BALL == pokemon.getPokeball())
+        	return "Ultra Ball";
+        else if (ItemId.ITEM_GREAT_BALL == pokemon.getPokeball())
+        	return "Great Ball";
+        else if (ItemId.ITEM_POKE_BALL == pokemon.getPokeball())
+        	return "Poke Ball";
+        else
+        	return "Hatched";
+    }
+
+    public String getPokeballImage() {
+        if (ItemId.ITEM_MASTER_BALL == pokemon.getPokeball())
+        	return "http://cdn.bulbagarden.net/upload//6/6d/Bag_Master_Ball_Sprite.png";  
+        else if (ItemId.ITEM_ULTRA_BALL == pokemon.getPokeball())
+        	return "http://cdn.bulbagarden.net/upload//0/03/Bag_Ultra_Ball_Sprite.png";
+        else if (ItemId.ITEM_GREAT_BALL == pokemon.getPokeball())
+        	return "http://cdn.bulbagarden.net/upload//c/ca/Bag_Great_Ball_Sprite.png";
+        else if (ItemId.ITEM_POKE_BALL == pokemon.getPokeball())
+        	return "http://cdn.bulbagarden.net/upload//9/93/Bag_Pok%C3%A9_Ball_Sprite.png";
+        else
+        	return "http://cdn.bulbagarden.net/upload/d/de/GO_Egg.png";
+    }
+        
+    public String getFavoriteIcon() {
+        if (pokemon.isFavorite() == true)
+        {
```

Comment:
At the same time, in the rest of the code a `{` comes right after `if` statement. Like:

```
if() {
} else {
}
```